contracts/root/Vesting.sol

```diff
@@ -25,13 +27,18 @@ contract Vesting is Ownable {
     mapping(address => uint256) public userPlanId;
     mapping(address => uint256) public allocations;
     mapping(address => uint256) public claimed;
+    mapping(address => uint256) public vtSQTAllocations;
```

Comment:
I feel we don't need a separated mapping? it should always equal to allocations

Reply:
when user withdraw `vtToken`, it will not, `allocations[user] >= vtSQTAllocations[user]`